docs/operator-manual/policy-evaluation-timeout.md

```diff
@@ -0,0 +1,184 @@
+
+When using the [`PolicyServer`](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/PolicyServer/v1@v1.4.2)
+Kurnetes Custom Resource Definition, these environment variables can be set in
+this way:
+
+```yaml
+# A Policy Server that has policy evaluation timeout disabled
+apiVersion: policies.kubewarden.io/v1
+kind: PolicyServer
+metadata:
+  name: no-policy-timeout
+spec:
+  env:
+  - name: KUBEWARDEN_DISABLE_TIMEOUT_PROTECTION
+    value: "true"
+---
+# A Policy Server that has policy evaluation timeout enabled,
+# with a 3 seconds timeout value
+apiVersion: policies.kubewarden.io/v1
+kind: PolicyServer
+metadata:
+  name: custom-policy-timeout
+spec:
+  env:
+  - name: KUBEWARDEN_POLICY_TIMEOUT
+    value: "3"
+```
+
+## Comparison with Kubernetes Dynamic Admission Controller timeout
+
+Kubewarden is implemented as Kubernetes [Dynamic Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/),
+which are implemented via [Webhooks](https://en.wikipedia.org/wiki/Webhook).
+
+Under the hood, Kubernetes API server makes a HTTP request against Policy Server
+describing an event that is about to happen. After the HTTP request is made,
+Kubernetes API Server waits for an answer to be provided. However, Kubernetes
+API server will not wait forever, after a certain amount of time it will
+consider the request to be "timed-out".
+
+Quoting the [official Kubernetes documentation](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts):
+
+> Default timeout for a webhook call is 10 seconds, You can set the timeout and
+> it is encouraged to use a short timeout for webhooks.
+> If the webhook call times out, the request is handled according to the
+> webhook's failure policy.
+
+That means that, regardless of the policy evaluation timeout feature, each
+Kubernetes admission request is subject to a timeout.
+
+Every Kubewarden Policy can set its own timeout value via the `timeoutSeconds`
+attribute of the `ClusterAdmissionPolicy` and `AdmissionPolicy` custom resources.
+By default the timeout value is 10 seconds.
+
+:::info
+
+All the Kubernetes admission requests made against a Policy Server are subject
+to two different timeouts:
+
+* The Kubernetes API server timeout value. Set to 10 seconds by default, tunable
+  on a per-policy basis via a dedicated attribute on the Kubewarden Custom Resources.
+* The Policy Server policy evaluation timeout
+
+:::
+
+Let's go through the following scenarios to better understand the differences
+between Kubernetes' Webhook timeout and Kubewarden's policy evaluation timeout.
+
+### Kubewarden policy evaluation timeout is disabled
+
+Let's assume we have a Policy Server that has the policy evaluation timeout
+feature disabled. This server is hosting a policy that is affected by a bug
+which causes an infinite loop to be entered.
```

Comment:
The infinite loop is during evaluation, per my understanding. However, we can stop the sentence at the word bug because we're explaining the same process below.
```suggestion
which causes it to enter an infinite loop during evaluation.
```